README.md

```diff
@@ -82,39 +81,11 @@ We plan to utilize API versioning to make automatic updates safe for consumer co
 
 To use the Block Data API after activation, skip to [Usage](#usage).
 
-### Install via `git subtree`
-
-We recommend this method for non-[WordPress VIP][wpvip] customers.
-
-The latest version of the VIP Block Data API plugin is available in the default `trunk` branch of this repository.
-
-We recommend installing the latest plugin version [via `git subtree`][wpvip-plugin-subtrees] within your site's repository:
-
-```bash
-# Enter your project's root directory:
-cd my-site-repo/
-
-# Add a subtree for the trunk branch:
-git subtree add --prefix plugins/vip-block-data-api git@github.com:Automattic/vip-block-data-api.git trunk --squash
-```
-
-To deploy the plugin to a remote branch, `git push` the committed subtree.
-
-The `trunk` branch will stay up to date with the latest version of the plugin. Use this command to pull the latest `trunk` branch changes:
-
-```bash
-git subtree pull --prefix plugins/vip-block-data-api git@github.com:Automattic/vip-block-data-api.git trunk --squash
-```
-
-Ensure that the plugin is up-to-date by pulling changes often.
-
-Note: We **do not recommend** using `git submodule`. [Submodules on WPVIP that require authentication][wpvip-plugin-submodules] will fail to deploy.
-
 ### Install via ZIP file
 
 The latest version of the plugin can be downloaded from the [repository's Releases page][repo-releases]. Unzip the downloaded plugin and add it to the `plugins/` directory of your site's GitHub repository.
 
-### Plugin activation
+#### Plugin activation
```

Comment:
I'd keep this at a level 3 header, as it makes "plugin activation" a child of "Install via ZIP file" structurally and in the table of contents.

Reply:
Discussed it, and will keep this change.